### ISSUE TITLE
!todo `per bend counterchanged` regression

### ISSUE THREAD

**Issue by karlwilcox** [drawshield/Drawshield-Code#266]
!todo `per bend counterchanged` regression
Examples of `... per bend counterchanged` that no longer work
https://drawshield.net/gallery/57/gallery-5706.html
https://drawshield.net/gallery/91/gallery-9153.html
https://drawshield.net/gallery/70/gallery-7020.html
https://drawshield.net/gallery/95/gallery-9546.html
https://drawshield.net/gallery/70/gallery-7016.html
https://drawshield.net/gallery/52/gallery-5270.html
https://drawshield.net/gallery/87/gallery-8783.html
https://drawshield.net/gallery/87/gallery-8782.html

**Comment by ericscheid** [comment]
For example, this: `Bendy sinister azure and ermine and per bend counterchanged` used to look like this (but now does not)
![image](https://user-images.githubusercontent.com/2605467/134112046-188079fc-fba2-4e67-98b0-24287f65778a.png)

**Comment by karlwilcox** [comment]
Yes, it looks like we've been a bit too clever for our own good here...

Compare:

_per pale azure and or  and  per fess vert and argent_

Which (correctly, if not strictly heraldically) produces:
![shield](https://user-images.githubusercontent.com/3171760/134259623-81639d88-2666-4529-89bd-d870b7a83d55.png)

with:


_per pale azure and or  and  per fess counterchanged_

Which is superficially similar but in fact means something completely different. Need to think about how to parse this properly....

**Comment by ericscheid** [comment]
More counterchanged confusion visible comparing 

- `crimson an escutcheon per bend sinister or and vert a boar sable`
- `crimson an escutcheon per bend sinister or and vert a boar counterchanged`
